helmfile/values/harbor.yaml.gotmpl

```diff
@@ -32,7 +32,11 @@ persistence:
       size: {{ .Values.harbor.redis.persistentVolumeClaim.size }}
   imageChartStorage:
     disableredirect: {{ .Values.harbor.persistence.disableRedirect }}
-    {{ if eq .Values.harbor.persistence.type "swift" }}
+    {{ if eq .Values.harbor.persistence.type "filesystem" }}
+    type: filesystem
+    filesystem:
+      rootdirectory: /storage
```

Comment:
How does this work? Will harbor mount this path from the host or something like that? Do we need to add "additional volumes"?

Reply:
We actually already have a volume for the registry (https://github.com/elastisys/compliantkubernetes-apps/blob/baaf7893c08afe0058a520f6f3bee8fb439fe551/helmfile/values/harbor.yaml.gotmpl#L25), and this seems to use the volume correctly.

Reply:
Oh I see. Filesystem apparently means "use a PVC" :smile: well I guess that is fine then :slightly_smiling_face:

Reply:
Yes, I agree that it's not very intuitive :smile: